website/docs/r/iam_resource_group.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory : "Account Management"
+---
+
+# ovh_iam_resource_group (Resource)
+
+Provides an OVHcloud IAM resource group.
+
+## Example Usage
+
+```hcl
+resource "ovh_iam_resource_group" "my_resource_group" {
+    name = "my_resource_group"
+    resources = [
+        "urn:v1:eu:resource:service1:service1-id",
+        "urn:v1:eu:resource:service2:service2-id",
+    ]
+}
+```
+
+## Argument Reference
+
+* `name`- Name of the resource group
+* `resources`- Set of the URNs of the resources contained in the resource group. All urns must be ones of valid resources
+
+## Attributes Reference
+
```

Comment:
- [ ] `resources` field is missing ?

Reply:
Resource is not an attribute but an argument, so it is described in the arguments sections.
Unless I am not understanfing you and you mean something else ?

Reply:
my bad, I read too quickly